File: prime.c
#include <stdio.h>
#include <math.h>

int is_prime(int num) {
    for (int i=2; i<(int)sqrt(num); i++) {
        if (num % i == 0)
            return 1;
    }
    return 0;
} 

int num_primes(int arrSize, int *numArr) {
    int count = 0;
    for (int i=0; i<arrSize; i++)
        count += is_prime(numArr[i]);
    return count;
}

File: prime.py
from typing import List

def is_prime(num: int):
    for i in range(2, int(num**(0.5))):
        if num % i == 0:
            return 1
    return 0

def num_primes(num_list: List[int]):
    count = 0
    for num in num_list:
        count += is_prime(num)
    return count
